src/main/java/com/pages/HomePage.java

```diff
@@ -0,0 +1,28 @@
+package com.pages;
+
+import com.microsoft.playwright.Page;
+
+public class HomePage extends Header {
+    Page page;
+
+    private final String mainPageHeaderLocator = "main>div.my-3>h1";
+    private final String createNewRepositoryButton = "article:nth-child(3) div:nth-child(1) > a";
```

Comment:
the same for selectors. Too unstable selectors. Do not use number. Only if it is table

Reply:
fixed